### ISSUE TITLE
The softmax Word2Vec method in TensorFlow

### ISSUE THREAD

**Issue by imsanjoykb** [imsanjoykb/Natural-Language-Processing#16]
As with any machine learning problem, there are two components – the first is getting all the data into a usable format, and the next is actually performing the training, validation and testing.  First I'll go through how the data can be gathered into a usable format, then we'll talk about the TensorFlow graph of the model.  Note that the code that I will be going through can be found in its entirety at this site's Github repository.  In this case, the code is mostly based on the TensorFlow Word2Vec tutorial here with some personal changes.
